interfaces/package.json

```diff
@@ -1,17 +1,19 @@
 {
   "name": "@binaris/shift-interfaces",
   "version": "0.1.2",
+  "version": "0.2.0-rc.0",
   "files": [
-    "dist"
+    "dist/"
   ],
   "description": "interfaces for serving ShiftJS",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rm -rf dist/ && tsc && npm run compile-schema",
-    "compile-schema": "typescript-json-schema -o dist/schema.json --required --noExtraProps tsconfig.json '*'",
+    "build": "rm -rf dist/ && tsc",
     "lint": "tslint -c ../common/tslint.yml -p .",
-    "test": "echo interface needs no tests"
+    "test": "echo interface needs no tests",
+    "build-koa-server": "rm -fr gen-server/ && concord node --server koa -o gen-server ${npm_package_name}-koa-server@${npm_package_version} src/index.ts",
```

Comment:
Is this required here?

Reply:
Why? these are duplicates of the wrapper package scripts.
These commands don't publish either.